frontend/app/src/components/price-manager/historic/HistoricPriceTable.vue

```diff
@@ -89,7 +88,12 @@ const deletePrice = async (item: HistoricalPrice) => {
 const fetchPrices = async (payload?: Partial<ManualPricePayload>) => {
   set(loading, true);
   try {
-    set(prices, await fetchHistoricalPrices(payload));
+    const filteredPayload = {
+      fromAsset: payload?.fromAsset || null,
+      toAsset: payload?.toAsset || null
```

Comment:
Because the fromAsset and toAsset can be empty string (''), but it is not allowed from backend. So I change it to null, so it will be filtered out later by `nonEmptyProperties`

Reply:
Can't we have a parametrized version of nonEmptyProperties where it also counts empty strings as empty properties instead?

At least for cases we are aware it seems like a cleaner approach to me.

Reply:
yeah it is makes sense.
updated now